internal/testidentity/revocation_db.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testidentity
+
+import (
+	"testing"
+
+	"github.com/alicebob/miniredis"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/peertls/extensions"
+	"storj.io/storj/storage"
+)
+
+// RevocationDBsTest runs the passed test function with each type of revocation database.
+func RevocationDBsTest(ctx *testcontext.Context, t *testing.T, test func(*testing.T, extensions.RevocationDB, storage.KeyValueStore)) {
```

Comment:
Each root-subtest probably should use a separate testcontext otherwise the files in the same folder can end up conflicting.